### DISCUSSION TITLE
Add text-input-v1 support

### DISCUSSION THREAD

**Discussion by ZenQy** [YaLTeR/niri#526] [Ideas]
support text-input-v1 can help to make fcitx5 work in electron apps on native wayland.
support in kde is here: [link](https://invent.kde.org/plasma/kwin/-/merge_requests/3403)
support in Hyprland is here: [link](https://github.com/hyprwm/Hyprland/pull/1778)

**Comment by YaLTeR** [comment]
Needs to be implemented in Smithay: https://github.com/Smithay/smithay/issues/1447
